src/pbt/deployment/jobs/airflow.py

```diff
@@ -422,36 +453,38 @@ def _deploy_add_jobs(self):
         for future in as_completed(futures):
             responses.append(future.result())
 
-        self._update_state(responses, self._operation_to_step_id[Operation.Add])
+        update_state(responses, self._operation_to_step_id[Operation.Add])
         return responses
 
     def _add_job(self, job_id, job_data):
         dag_name = job_data.dag_name
         zipped_dag_name = get_zipped_dag_name(dag_name)
         zip_folder(self._project.load_airflow_folder(job_id), zipped_dag_name)
-
+        fabric_name = self._project_config.fabric_config.get_fabric(job_data.fabric_id).name
         client = self.get_airflow_client(fabric_id=job_data.fabric_id)
         try:
             client.upload_dag(dag_name, zipped_dag_name)
             try:
                 client.unpause_dag(dag_name)
-                log(f"Successfully un-pause dag {dag_name} for job {job_id} and fabric {job_data.fabric_id}",
+                log(f"{Colors.OKGREEN}Successfully un-pause dag `{dag_name}` for job `{job_id}` and fabric-id {job_data.fabric_id}{Colors.ENDC}",
+                    step_id=self._ADD_JOBS_STEP_ID, level=LogLevel.TRACE)
+                log(f"{Colors.OKGREEN}Successfully un-paused dag `{dag_name}` for job `{job_id}` and fabric `{fabric_name}`{Colors.ENDC}",
```

Comment:
would it be easier to create one log method per color instead of appending for every log call?